src/pages/Product/components/Scroller/Scroller.css

```diff
@@ -0,0 +1,20 @@
+.scroller-container img {
+    width: 100%;
+    height: 370px;
+}
```

Comment:
remove height - let images be their natural height

Reply:
rdy